Test/Test.cs

```diff
@@ -146,164 +46,480 @@ public void InitClient()
+            var transitionId = $"las:transition:{Guid.NewGuid().ToString()}";
+            var response = Toby.GetTransitionExecution(transitionId, executionId);
+            CheckKeys(new [] {"transitionId", "executionId", "status"}, response);
+        }
+
+        [TestCase("foo", "bar")]
+        [TestCase(null, null)]
+        public void TestUpdateTransition(string name, string description) {
+            var schema = new Dictionary<string, string>() {
+                {"schema", "https://json-schema.org/draft-04/schema#"},
+                {"title", "response"}
+            };
+            var inputSchema = schema;
+            var outputSchema = schema;
+            var transitionId = $"las:transition:{Guid.NewGuid().ToString()}";
+            var response = Toby.UpdateTransition(transitionId, inputSchema, outputSchema, name, description);
+            CheckKeys(new [] {"name", "transitionId", "transitionType"}, response);
+        }
+
+        [Test]
+        public void TestExecuteTransition() {
+            var transitionId = $"las:transition-execution:{Guid.NewGuid().ToString()}";
+            var response = Toby.ExecuteTransition(transitionId);
+            CheckKeys(new [] {"transitionId", "executionId", "status"}, response);
+        }
+
+        [TestCase(
+            "running",
+            "las:transition-execution:08b49ae64cd746f384f05880ef5de72f",
+            3,
+            null,
+            "completedBy",
+            "asc"
+        )]
+        public void TestListTransitionExecutions(
+            string? status = null,
+            string? executionId = null,
+            int? maxResults = null,
+            string? nextToken = null,
+            string? sortBy = null,
+            string? order = null
+        ) {
+            var transitionId = $"las:transition:{Guid.NewGuid().ToString()}";
+            var response = Toby.ListTransitionExecutions(
+                transitionId,
+                new List<string>{ status },
+                new List<string>{ executionId },
+                maxResults,
+                nextToken,
+                sortBy,
+                order
+            );
+            var expectedKeys = new [] {"executions"};
+            CheckKeys(expectedKeys, response);
+        }
+
+        [TestCase(
+            3,
+            null,
+            "completedBy",
+            "asc"
+        )]
+        public void TestListTransitionExecutions(
+            int? maxResults = null,
+            string? nextToken = null,
+            string? sortBy = null,
+            string? order = null
+        ) {
+            var transitionId = $"las:transition:{Guid.NewGuid().ToString()}";
+            var statuses = new List<string>{ "running", "succeeded" };
+            var executionIds = new List<string>{
+                $"las:execution:{Guid.NewGuid().ToString()}",
+                $"las:execution:{Guid.NewGuid().ToString()}"
```

Comment:
"las:transition-execution:<uuid>" are valud execution-ids, these values should yield an error.